### PR TITLE
Added 🤗  model and spaces links

### PR DESCRIPTION
**Pull request by vrindaprabhu**
As per the ongoing 🤗 keras sprint, have updated the following examples:

Vision:

- metric_learning 
- visualizing_what_convnets_learn

NLP:

- addition_rnn 

Will be glad to address any issues/changes.

### CODE REVIEW

**Comment by merveenoyan** [comment]
Hello Vrinda, 
So you either only add a notebook and then generate the python and markdown file OR you add python file and generate the notebook and markdown. On README.md of the repo, you will find how to get a local build of keras.io website and you can check it out. it's good to generate one from another because it allows you to be consistent. 🙂 The steps are well-described on README on how to generate the example and (then) get a local build. 
If you're changing the notebook, make sure you add least amount of change (you can see the diff, because sometimes the cell metadata is affected due to versions on your local, so I highly suggest you to convert from a python script instead)
Feel free to ping me btw if you have any problems, I'm super-happy to help.

File: examples/nlp/ipynb/addition_rnn.ipynb [hunk 1]
```diff
@@ -330,7 +328,13 @@
     "colab_type": "text"
    },
    "source": [
-    "You'll get to 99+% validation accuracy after ~30 epochs.\n"
+    "You'll get to 99+% validation accuracy after ~30 epochs.\n",
+    "\n",
+    "Example available in HuggingFace.\n",
```
Comment (fchollet) [inline examples/nlp/ipynb/addition_rnn.ipynb:333]: in -> on

File: examples/vision/ipynb/metric_learning.ipynb [hunk 2]
```diff
@@ -488,7 +488,20 @@
...
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text"
+   },
+   "source": [
+    "Example available in HuggingFace.\n",
```
Comment (fchollet) [inline examples/vision/ipynb/metric_learning.ipynb:500]: Likewise

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**